lib/SelectableItem/SelectableItem.less

```diff
@@ -3,20 +3,15 @@
 	padding-left: (@moon-item-indent + @moon-spotlight-outset);
 
 	&:before {
-		content: '';
+		content: '\2022';
```

Comment:
Change this line to be `content: '\0F0013';` and add: `font-family: "Moonstone Icons";`
Using the circle glyph from Moonstone Icons will generate consistent results on multiple platforms with various fonts installed.